### ISSUE TITLE
Sign In Button is not displayed

### ISSUE THREAD

**Issue by wieblinger1** [firebase/friendlychat#103]
I'm in step 8 and the three sample messages and the sign in button don't show up. I can only see the same as the picture in step 6. I'm in the tutorial: https://codelabs.developers.google.com/codelabs/firebase-web

**Comment by EasonKao** [comment]
Same issue, please help.

**Comment by nicolasgarnier** [comment]
Hey both of you. Could you check the Javascript console and see if there are any errors displayed there? Also could you depoy your code online (using `firebase deploy`) and give me the URL?

**Comment by EasonKao** [comment]
I found the solution in the web folder, just need to add a source script "main.js" in index page as below:
`<script src="https://www.gstatic.com/firebasejs/3.5.1/firebase.js"></script>
<script>
  // Initialize Firebase
  var config = {
    apiKey: "xxx",
    authDomain: "xxx",
    databaseURL: "xxx",
    storageBucket: "xxx",
    messagingSenderId: "xxx"
  };
  firebase.initializeApp(config);
</script>
<script src="scripts/main.js"></script>`

**Comment by jbarth04** [comment]
I'm having a similar issue on step 7 of FriendlyChat.  I've added the specified code to index.html  and main.js.  When I deploy the app using "firebase deploy", the sign-in with google button appears.  However, when I try to serve the page locally (on http://localhost:5000), the google sign-in button does not appear.  I am using Chrome and I do not see any errors in the Javascript console.  I have turned off my ad-blocker.  When I run "npm run build" (as suggested on another post), I do get an npm error that package.json does not exist.  Any suggestions why the sign in button does not work locally?  Thanks in advance!
